### PR TITLE
Csi sanity fix

### PR DESCRIPTION
**Pull request by hemalathagajendran**
## Pull request checklist

```release-notes

```

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [x] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## What is the current behavior?

-

## What is the new behavior?

-

## How risky is this change?
- [ ] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Review by deeghuge — CHANGES_REQUESTED** [review]
@hemalathagajendran the space is getting added in front of all comments. Your IDE might be adding it by default.

**Comment by hemalathagajendran** [comment]
> @hemalathagajendran the space is getting added in front of all comments. Your IDE might be adding it by default.

Removed extra spaces which is added from IDE

**Comment by hemalathagajendran** [comment]
> @hemalathagajendran the space is getting added in front of all comments. Your IDE might be adding it by default.

Done

File: driver/csiplugin/controllerserver.go [hunk 1]
```diff
@@ -604,7 +609,7 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 			srcVolumeIDMembers, err = getVolIDMembers(srcVolumeID)
 			if err != nil {
 				glog.Errorf("volume:[%v] - Invalid Volume ID %s [%v]", volName, srcVolumeID, err)
-				return nil, err
+				return nil, status.Error(codes.NotFound, "volume source volume is not found")
```
Comment (deeghuge) [inline driver/csiplugin/controllerserver.go:612]: Lets keep err in return as well

**Comment by Jainbrt** [comment]
@hemalathagajendran looks like few changes are being pushed using root user, we should avoid that. Either we should revert them or raise a new PR without the root use checkin

File: driver/csiplugin/controllerserver.go (hunk 1 above)
Comment (hemalathagajendran) [reply driver/csiplugin/controllerserver.go:612]: Done changes in  new PR https://github.com/IBM/ibm-spectrum-scale-csi/pull/842

File: driver/csiplugin/controllerserver.go [hunk 2]
```diff
@@ -615,7 +620,7 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 				snapIdMembers, err = cs.GetSnapIdMembers(snapId)
 				if err != nil {
 					glog.Errorf("volume:[%v] - Invalid snapshot ID %s [%v]", volName, snapId, err)
-					return nil, err
+					return nil, status.Error(codes.NotFound, "volume source snapshot is not found")
```
[the remark this answers is not in the document]
Comment (hemalathagajendran) [reply driver/csiplugin/controllerserver.go:623]: Done changes in new PR https://github.com/IBM/ibm-spectrum-scale-csi/pull/842

**Comment by hemalathagajendran** [comment]
> @hemalathagajendran looks like few changes are being pushed using root user, we should avoid that. Either we should revert them or raise a new PR without the root use checkin

Raised new PR https://github.com/IBM/ibm-spectrum-scale-csi/pull/842

**Pull request closed without merging**